crds/tests/test_certify.py

```diff
@@ -800,6 +830,15 @@ def certify_rmap_compare():
     >>> test_config.cleanup(old_state)
     """
 
+def certify_roman_rmap_compare():
+    """
+    >>> old_state = test_config.setup(url="https://roman-crds-serverless.stsci.edu", observatory="roman", cache=test_config.CRDS_TESTING_CACHE)
+    >>> certify.certify_file("roman_wfi_flat_0004.rmap", context="roman_0004.pmap")
+    CRDS - INFO -  Certifying 'roman_wfi_flat_0004.rmap' as 'MAPPING' relative to context 'roman_0004.pmap'
+    >>> test_config.cleanup(old_state)
+    """
+
+
 def certify_jwst_bad_fits():
```

Comment:
Do we really want certify tests for all missions in the same file?

Reply:
I think I was the one who made the suggestion to work in this file, but I see the appeal of isolating the Roman tests to their own module.  Besides helping us stay organized as we add more and more missions in here, it would also let us run everything Roman in one go.

Reply:
I like moving these to their own file as well, but I think it needs to be its own issue / PR, as it requires moving Ed's tests and recreating the class structure, thus taking it beyond the scope of this PR.

Reply:
Issue made.